test/unit/BondingManager.js

```diff
@@ -4,6 +4,7 @@ import {contractId, functionSig, functionEncodedABI} from "../../utils/helpers"
 import {constants} from "../../utils/constants"
 import BN from "bn.js"
 import truffleAssert from "truffle-assertions"
+import {assert} from "chai"
```

Comment:
Looks like this was re-added by vscode plugins upon saving?  I don't mind leaving it there tbh as it's the actual dependency that's being injected anyway.